typings/market-closure/index.d.ts

```diff
@@ -0,0 +1,47 @@
+declare module 'market-closure' {
```

Comment:
Later you can migrate your lib to TS so we get types for free and won't need to declare them here.